_posts/2024-02-05-cde-customization.adoc

```diff
@@ -0,0 +1,434 @@
+---
+title: Customizing Eclipse Che Cloud Development Environments
+layout: post
+author: Mario Loriedo
+description: >-
+  In this article we review the different ways to customize an Eclipse Che Cloud Development Environment
+categories: []
+keywords: ['CDE']
+slug: /@mario.loriedo/cde-customization
+---
+
+Eclipse Che provides Cloud Development Environments (CDEs) with a click. The default container of those CDEs, what we call the https://github.com/devfile/developer-images/[universal developer image], has a lot of pre-installed development tools but may not have the right version or the right tool for you. If that's the case, no worries: one Eclipse Che CDEs main assets is configurability! 
+
+In this blog posts we will review Eclipse Che CDEs customization in 4 parts:
+
+<<part1>>
+  
+<<part2>>
+
+<<part3>>
+
+<<part4>>
+
+
+NOTE: To help illustrate this sample we will use a https://github.com/l0rd/rails-sample[Ruby on Rails source code example]. Ruby is a good example because the default development container doesn't include Ruby development tools at all. For other languages such as Java, Go, NodeJS, Python and .NET, Eclispe Che default development image includes a few tools, and the customizations in this blog post may not be necessary.
```

Comment:
```suggestion
NOTE: To help illustrate this sample we will use a https://github.com/l0rd/rails-sample[Ruby on Rails source code example]. Ruby is a good example because the default development container doesn't include Ruby development tools at all. For other languages such as Java, Go, NodeJS, Python and .NET, the Eclipse Che default development image includes a few tools, and the customizations in this blog post may not be necessary.
```